File: csharp/AlphabetTriangle.cs
using System;

class AlphabetTriangle {
    static void Main(string[] args) {
        int n = 5;
        string alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                Console.Write(" ");
            }
            for (int j = 0; j <= i; j++) {
                Console.Write(alphabet[j]);
            }
            Console.WriteLine();
        }
    }
}

File: java/AlphabetTriangle.java
public class AlphabetTriangle {
    public static void main(String[] args) {
        int n = 5;
        String alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j <= i; j++) {
                System.out.print(alphabet.charAt(j));
            }
            System.out.println();
        }
    }
}
